### DISCUSSION TITLE
Breadcrumbs for hierarchical collection

### DISCUSSION THREAD

**Discussion by lasseron** [statamic/cms#5174] [Q&A]
Hi! I have a documentation site which uses just one hierarchical collection for both the topics (tree) and the help article itself (leaf).
The routes are in format of "help/{slug}". 

I wish to display a breadcrumbs on both topic and article pages which should indicate the whole tree eg
Help > Topic > Subtopic > Article title.

I tried nav:breadcrumbs but it seems to rely on the route format and does not return the whole tree for me.

Then I tried to iterate through all the parents using "parent" but I need to detect the current level in the tree somehow.

Perhaps there is more elegant way to get the breadcrumbs or can anybody hint me about how to find the level of a node in collection tree.

**Comment by edalzell** [comment]
use the `nav` tag maybe?

**Reply by lasseron** [reply]
My tree has multiple levels such as Help > Topic > Subtopic > Article.
But the URL-s are in format of help/current-level-slug.
So for example if I'm viewing the Article, the slug is help/article-title and the nav tag only returns two elements. However I need the whole tree of Help > Topic > Subtopic > Article.

**Reply by edalzell** [reply]
are you using the `children` part of the nav tag?

**Reply by lasseron** [reply]
I tried the example `nav:breadcrumbs` code from https://statamic.dev/tags/nav-breadcrumbs. 
This does work only if the number of url segments is the same as the depth of the hierarchy. 

As about the `nav` tag, I'm not sure how to traverse the tree from bottom to up, in other words from leaf to tree root.

Just to rephrase my question, I have a hierarchical collection:
<img width="520" alt="image" src="https://user-images.githubusercontent.com/238244/153881753-f0fe6c47-7048-4f9f-b094-0ffc60c898e9.png">

The url of the post on the bottom of the image is 
help/my-credit-card-payment-failed.

I wish I could display breadcrumbs:
Help > For ticket buyers > Frequently asked questions > My credit card payment failed
